docker/quickstart-flink/prepare_build.sh

```diff
@@ -0,0 +1,183 @@
+            return 1
+        fi
+        log_success "Checksum verified for $description"
+    else
+        log_success "Downloaded $description"
+    fi
+}
+
+# Check if required directories exist
+check_prerequisites() {
+    log_info "Checking prerequisites..."
+
+    local required_dirs=(
+        "$PROJECT_ROOT/fluss-flink/fluss-flink-1.20/target"
+        "$PROJECT_ROOT/fluss-lake/fluss-lake-paimon/target"
+        "$PROJECT_ROOT/fluss-flink/fluss-flink-tiering/target"
+    )
+
+    for dir in "${required_dirs[@]}"; do
+        if [ ! -d "$dir" ]; then
+            log_error "Required directory not found: $dir"
+            log_error "Please build the Fluss project first: mvn clean package"
+            exit 1
+        fi
+    done
+
+    log_success "All prerequisites met"
+}
+
+# Main execution
+main() {
+    log_info "Preparing JAR files for Fluss Quickstart Flink Docker..."
+    log_info "Project root: $PROJECT_ROOT"
+
+    # Check prerequisites
+    check_prerequisites
+
+    # Clean and create directories
+    log_info "Setting up directories..."
+    rm -rf lib opt
+    mkdir -p lib opt
+
+    # Copy Fluss connector JARs
+    log_info "Copying Fluss connector JARs..."
+    copy_jar "$PROJECT_ROOT/fluss-flink/fluss-flink-1.20/target/fluss-flink-1.20-*.jar" "./lib" "fluss-flink-1.20 connector"
+    copy_jar "$PROJECT_ROOT/fluss-lake/fluss-lake-paimon/target/fluss-lake-paimon-*.jar" "./lib" "fluss-lake-paimon connector"
+
+    # Download external dependencies
+    log_info "Downloading external dependencies..."
+
+    # Download flink-faker for data generation
+    download_jar \
+        "https://github.com/knaufk/flink-faker/releases/download/v0.5.3/flink-faker-0.5.3.jar" \
+        "./lib/flink-faker-0.5.3.jar" \
+        "" \
+        "flink-faker-0.5.3"
+
+    # Download flink-shaded-hadoop-2-uber for Hadoop integration
+    download_jar \
+        "https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-hadoop-2-uber/2.8.3-10.0/flink-shaded-hadoop-2-uber-2.8.3-10.0.jar" \
+        "./lib/flink-shaded-hadoop-2-uber-2.8.3-10.0.jar" \
+        "5dd57b5d38965c0f70e3f63d2581755df6c296bb" \
+        "flink-shaded-hadoop-2-uber-2.8.3-10.0"
+
+    # Download paimon-flink connector
+    download_jar \
+        "https://repo1.maven.org/maven2/org/apache/paimon/paimon-flink-1.20/1.2.0/paimon-flink-1.20-1.2.0.jar" \
+        "./lib/paimon-flink-1.20-1.2.0.jar" \
+        "b9f8762c6e575f6786f1d156a18d51682ffc975c" \
+        "paimon-flink-1.20-1.2.0"
+
+    # Prepare lake tiering JAR
+    log_info "Preparing lake tiering JAR..."
+    copy_jar "$PROJECT_ROOT/fluss-flink/fluss-flink-tiering/target/fluss-flink-tiering-*.jar" "./opt" "fluss-flink-tiering"
+
+    # Summary
+    log_success "JAR files preparation completed!"
+    echo ""
+    log_info "📦 Generated JAR files:"
+    echo "Lib directory:"
+    ls -la ./lib/ 2>/dev/null || echo "  (empty)"
+    echo "Opt directory:"
+    ls -la ./opt/ 2>/dev/null || echo "  (empty)"
```

Comment:
Could we add a final check for the copied jars under lib and opt to guarantee there is no missing jars there?